### ISSUE TITLE
It is possible to lose Guthbered's ring without advancing the quest while talking to Harlenn

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#218]
```
Originally reported by Hecubah:
http://andors.techby2guys.com/viewtopic.php?f=2&t=823

After defeating Guthbered, and delivering the ring to Harlenn, it is possible 
to lose the quest item (Guthbered's ring) without advancing the quest, thus 
getting stuck in the quest with no way to advance it. This is possible if you 
chose to exit conversation right after you have given the ring to Harlenn.

Instead, the quest should advance as soon as you give the ring to Harlenn, so 
that you can exit dialogue and return later to continue the conversation.

This bug is also present in the corresponding dialogue of Guthbered (same 
error).
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 16 Jul 2011 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
This is now fixed in r164.
```

Original comment by `oskar.wi...@gmail.com` on 2 Oct 2011 at 7:35
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
